src/components/refactor/rhf-inputs/integer-input.js

```diff
@@ -8,8 +8,9 @@
 import React from 'react';
 import TextInput from './text-input';
 import PropTypes from 'prop-types';
+import { toPositiveIntValue } from 'components/dialogs/dialogUtils';
 
-const IntegerInput = (props) => {
+const IntegerInput = (props, isPositive) => {
```

Comment:
```suggestion
const IntegerInput = ({isPositive = false, ...props}) => {
OR
const IntegerInput = (props) => {
    const {isPositive = false} = props;
```

Also, I think I would rename "isPositive" to "isInputPositiveOnly", or somethink like that

Reply:
Done